packages/@lwc/engine-core/src/framework/rendering.ts

```diff
@@ -604,12 +603,7 @@ function patchElementPropsAndAttrs(
     patchClassAttribute(oldVnode, vnode, renderer);
     patchStyleAttribute(oldVnode, vnode, renderer);
 
-    if (vnode.data.external) {
-        patchAttrUnlessProp(oldVnode, vnode, renderer);
-    } else {
-        patchAttributes(oldVnode, vnode, renderer);
-    }
```

Comment:
Assuming the `external` const is hoisted out of the loop, yeah I imagine it's not that expensive. I can run the benchmarks to confirm though.

Reply:
Actually can we just run the benchmarks at the end of the release like usual? It would take a long time (~3 hours) to run the full benchmarks to confirm no regression here, and I don't think it's worth it.